### DISCUSSION TITLE
How to resolve Golang golang.org/x/net and google.golang.org/grpc packages vulnerability

### DISCUSSION THREAD

**Discussion by dyegocruz** [vercel/turborepo#6379] [Help] [answered]
After using the Trivy Scan on my project, Even using the last turbo lib version, I have the result below.

<img width="1259" alt="image" src="https://github.com/vercel/turbo/assets/872297/3804ad27-3f9f-4633-ae8a-65ec531ed9da">

How can I resolve this?

**Comment by anthonyshew** [comment, marked as answer]
We're completing a port to Rust from Golang soon so these dependencies will be gone entirely shortly! Stay tuned!
